components/AdForm/types.js

```diff
@@ -1,27 +1,48 @@
 import { range } from 'lodash';
 
-export const listPositions = ['before post 1'].concat(
-  range(1, 30).map(n => `after post ${n}`),
-);
+const MIN_LIMIT_VALUE = 300; // This must be equal to this: https://github.com/frontity/saturn-theme/blob/dev/src/shared/components/HtmlToReactConverter/injectSlots.js#L6
 
-export const singlePositions = ['before paragraph 1'].concat(
-  range(1, 30).map(n => `after paragraph ${n}`),
-);
+export const positions = {
```

Comment:
@luisherranz this file was already changed in the previous pull request, I don't know if this might provoke any conflicts.

Reply:
"This branch has no conflicts with the base branch" <- not in theory

Reply:
Oh, I just remembered I rebased this branch onto the other to avoid precisely that.